gapis/api/gles/api/draw_commands.api

```diff
@@ -452,6 +452,31 @@ sub void DrawCommandDependencies(ref!Context ctx) {
 
   }
 
+  // Vertex input
+  if !IsDefaultVertexArrayBound() {
+    // Read host-side vertex input data. glDraw*() typically calls ReadVertexArrays() which
+    // reads client-side vertex input data, the only necessary reads at capture time.
+    // Here the reads for host-side vertex input is done to create dependencies.
+    arrayBuffer := ctx.Bound.ArrayBuffer
+    read(arrayBuffer.Data)
+    for _, _, a in ctx.Bound.VertexArray.VertexAttributeArrays {
+      if a.Enabled == GL_TRUE {
```

Comment:
In general are all of these read in all circumstances?
I don't know gles well but are a.Integer and a.Pointer both read every time?

Reply:
Yes, as far as I understand. The names may be misleading here, Integer is a boolean that indicates whether vertex attrib array has unconverted integers.